resources/zib-MedicationContraIndication.xml

```diff
@@ -0,0 +1,178 @@
+  <fhirVersion value="4.0.1" />
+  <mapping>
+    <identity value="zib-medicationcontraindication-v1.0-2020EN" />
+    <uri value="https://zibs.nl/wiki/MedicationContraIndication-v1.0(2020EN)" />
+    <name value="zib MedicationContraIndication-v1.0(2020EN)" />
+  </mapping>
+  <kind value="resource" />
+  <abstract value="true" />
+  <type value="Flag" />
+  <baseDefinition value="http://hl7.org/fhir/StructureDefinition/Flag" />
+  <derivation value="constraint" />
+  <differential>
+    <element id="Flag">
+      <path value="Flag" />
+      <short value="MedicationContraIndication" />
+      <comment value="A flag is a warning or notification of some kind presented to the user - it could be a physician or any other person involved in patient care. It usually represents something of sufficient significance to warrant a special representation of some kind - rather than just a note in the source. A flag has a subject that represents the resource that will trigger its display.&#xD;&#xA;&#xD;&#xA;The zib MedicationContraIndication defines no status. However, the concepts ReasonClosure and EndDate relate to the Flag inactive status. Therefore it is expected that status gets a matching value if ReasonClosure is present or the EndDate is in the past." />
+      <alias value="MedicatieContraIndicatie" />
+      <mapping>
+        <identity value="zib-medicationcontraindication-v1.0-2020EN" />
+        <map value="NL-CM:9.14.1" />
+        <comment value="MedicationContraIndication" />
+      </mapping>
+    </element>
+    <element id="Flag.extension">
+      <path value="Flag.extension" />
+      <slicing>
+        <discriminator>
+          <type value="value" />
+          <path value="url" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+    </element>
+    <element id="Flag.extension:comment">
+      <path value="Flag.extension" />
+      <sliceName value="comment" />
+      <type>
+        <code value="Extension" />
+        <profile value="http://nictiz.nl/fhir/StructureDefinition/ext-Comment" />
+      </type>
+    </element>
+    <element id="Flag.extension:comment.value[x]">
+      <path value="Flag.extension.value[x]" />
+      <short value="Comment" />
+      <definition value="Comment on the contraindication important for interpretation which cannot be captured in any of the other data items. For example, a comment on a contraindication with an end date in the future." />
+      <alias value="Toelichting" />
+      <mapping>
+        <identity value="zib-medicationcontraindication-v1.0-2020EN" />
+        <map value="NL-CM:9.14.7" />
+        <comment value="Comment" />
+      </mapping>
+    </element>
+    <element id="Flag.extension:reasonClosure">

```

Comment:
Is het nog een idee om deze extensie onder status te plaatsen? Het kan gezien worden als toelichting op de reden dat status _inactive_ is. @ArdonToonstra  wat vind jij?

Reply:
Hier hebben we een vraag over gesteld: https://bits.nictiz.nl/browse/ZIB-1347

Volgens de zib (Fred) zijn deze twee dingen niet direct gekoppeld met elkaar.

Reply:
Dan klopt het commentaar in het profiel niet.

Reply:
We hebben een verder antwoord op het ticket. Volgens mij kunnen we er wel vanuit gaan dat ReasonClosure normaliter enkel gegeven wordt wanneer de MedicatieContraindicatie _inactief_ is.

Ik ben geen voorstander om dit als extensie onder status te plaatsten. De ReasonClosure informatie zegt iets over de gehele Flag en niet alleen over de status. FHIR modeleert elementen die iets zeggen over de status ook naast elkaar en niet onder elkaar. Zie bijv. MedicationRequest.statusReason.

Reply:
Goed, het nu redelijk defensies opgeschreven ("expected"), dus verder akkoord.